src/gondi.ts

```diff
@@ -442,10 +442,16 @@ export class Gondi {
     return Number(result.nft.id);
   }
 
-  async collections(props: { statsCurrency?: Address }) {
-    const result = await this.api.collections({ currency: props.statsCurrency ?? zeroAddress });
-    const { edges: collections, pageInfo } = result.collections;
-    return { collections: collections.map((edge) => edge.node), pageInfo };
+  async collections(props: {
+    statsCurrency?: Address;
+    standards?: TokenStandardType[];
+    collections?: number[];
```

Comment:
are this ids or what? Confusing to me

Reply:
yes, they are ids. The query can filter only by that.